packages/components/pie-checkbox-group/README.md

```diff
@@ -100,6 +98,13 @@ In your markup or JSX, you can then use these to set the properties for the `pie
 </PieCheckboxGroup>
 ```
 
+## Slots
+
+| Slot | Description |
+|---|---|
+| `default` | Pass PieCheckbox components as direct children for the CheckboxGroup. |
+| `label` | Pass PieFormLabel to render the checkbox group label. |
```

Comment:
Just curious – why make this a slot, rather than a prop that automatically puts the string in a `pie-form-label` component?